Examples/US_Plastic_Exports.jl

```diff
@@ -0,0 +1,48 @@
+
+import HTTP
+using Dashboards
+
+external_stylesheets = ["https://codepen.io/chriddyp/pen/bWLwgP.css"]
+
+app = Dash("Plastic Exports", external_stylesheets=external_stylesheets) do
+    html_div() do
+        html_h2("Dashboards.jl interactive graph",
+            style=(
+                textAlign = "center",
+                color = "#8c8c8c"
+            )
+        ),
+
+
+        dcc_graph(
+            id = "my-graph",
+
+            figure = (
+                data = [
+                (x=[1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012],

```

Comment:
Can you reference where this data is from?

Reply:
I just commented on the task on the dashboard as well to let @NandVinchhi know this